.github/workflows/publish.yml

```diff
@@ -62,12 +62,4 @@ jobs:
         env:
           HELM_GCS_CREDENTIALS: ${{ secrets.HELM_GCS_CREDENTIALS }}
           HELM_GCS_REPOSITORY: ${{ secrets.HELM_GCS_REPOSITORY }}
-        run: |
-            CHART_VERSION=$(echo ${GITHUB_REF} | cut -d/ -f 3)
-            CHART_NAME=$(awk '/^name:/ {print $2}' ./helm/Chart.yaml)
-            export GOOGLE_APPLICATION_CREDENTIALS=${HOME}/helm-gcs-key.json
-            echo $HELM_GCS_CREDENTIALS > ${GOOGLE_APPLICATION_CREDENTIALS}
-            helm dependency update ./helm/
-            helm repo add helm-gcs $HELM_GCS_REPOSITORY
-            helm package --version ${CHART_VERSION} --app-version ${CHART_VERSION} ./helm/
-            helm gcs push ${CHART_NAME}-${CHART_VERSION}.tgz helm-gcs --public --retry
+        uses: hypertrace/actions/helm-gcs-publish@main
```

Comment:
Can we parameterize these two args for this action

Reply:
Actually, maybe not even needed - since this is our own action, it can probably reference the org secret directly, right?

Reply:
yup. It can refer org secret directly

Reply:
Do you want to update that before merging or handle separately? Will approve, up to you.

Reply:
Let's handle that separately if required. Let's cut the release post merge and see if there are any other issues.